<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFramedebugger</class>
 <widget class="QMainWindow" name="QtFramedebugger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame Debugger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="prevKickbutton">
            <property name="text">
             <string>Prev Draw Kick</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextKickButton">
            <property name="text">
             <string>Next Draw Kick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="context_1">
        <attribute name="title">
         <string>Context 1</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="context0_layout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSplitter" name="splitter_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTextEdit" name="contextState0">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Options</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QPushButton" name="saveContext1">
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="fitContext1Button">
                  <property name="text">
                   <string>Fit</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="context0Source">
                  <item>
                   <property name="text">
                    <string>Framebuffer</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Base)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 1)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 2)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 3)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 4)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 5)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 6)</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="context0Buffer"/>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="context_2">
        <attribute name="title">
         <string>Context 2</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="context1_layout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSplitter" name="splitter_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSplitter" name="splitter_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTextEdit" name="contextState1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Options</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QPushButton" name="saveContext2">
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="fitContext2Button">
                  <property name="text">
                   <string>Fit</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="context1Source">
                  <item>
                   <property name="text">
                    <string>Framebuffer</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Base)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 1)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 2)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 3)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 4)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 5)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Texture (Mip 6)</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="context1Buffer"/>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="inputStateTab">
        <attribute name="title">
         <string>InputState</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTextEdit" name="inputStateTextEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="vu1MicroprogramTab">
        <attribute name="title">
         <string>VU1 Microprogram</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_10"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1156</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Dump"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuFramebuffer_Display_Mode">
     <property name="title">
      <string>Framebuffer Display Mode</string>
     </property>
     <addaction name="actionRaw"/>
     <addaction name="action640x448_Non_Interlaced"/>
     <addaction name="action640x448_Interlaced"/>
    </widget>
    <widget class="QMenu" name="menuGS_Handler">
     <property name="title">
      <string>GS Handler</string>
     </property>
     <addaction name="actionGsHandlerOpenGL"/>
     <addaction name="actionGsHandlerVulkan"/>
    </widget>
    <addaction name="actionAlpha_Test_Enabled"/>
    <addaction name="actionDepth_Test_Enabled"/>
    <addaction name="actionAlpha_Blend_Enabled"/>
    <addaction name="menuFramebuffer_Display_Mode"/>
    <addaction name="menuGS_Handler"/>
   </widget>
   <widget class="QMenu" name="menuVirtual_Machine">
    <property name="title">
     <string>Virtual Machine</string>
    </property>
    <addaction name="actionStep_VU1"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuVirtual_Machine"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Dump">
   <property name="text">
    <string>Load Frame Dump...</string>
   </property>
  </action>
  <action name="actionAlpha_Test_Enabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alpha Test Enabled</string>
   </property>
  </action>
  <action name="actionDepth_Test_Enabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth Test Enabled</string>
   </property>
  </action>
  <action name="actionAlpha_Blend_Enabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alpha Blend Enabled</string>
   </property>
  </action>
  <action name="actionRaw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw</string>
   </property>
  </action>
  <action name="action640x448_Non_Interlaced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>640x448 Non-Interlaced</string>
   </property>
  </action>
  <action name="action640x448_Interlaced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>640x448 Interlaced</string>
   </property>
  </action>
  <action name="actionStep_VU1">
   <property name="text">
    <string>Step VU1</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionGsHandlerOpenGL">
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
    <string>OpenGL</string>
   </property>
  </action>
  <action name="actionGsHandlerVulkan">
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
    <string>Vulkan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
